library/testing/CMakeLists.txt

```diff
@@ -42,6 +41,13 @@ if(F3D_MODULE_UI)
   endif()
 endif()
 
+# This test fails with vtk 9.2.6
```

Comment:
why is it failing now?

Reply:
no idea tbh, I can investigate though

Reply:
Since we now delete timer in the timer callback we need this fix:
https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10155